dashboard/api.py

```diff
@@ -518,12 +518,9 @@ def get_exam_register_end_date(course):
     """
     Get a formatted string of dates during which the exam is schedulable
     """
-    now = now_in_utc()
-    schedulable_exam_run = ExamRun.objects.filter(
-        course=course, date_last_eligible__gte=now.date()
-    ).first()
+    schedulable_exam_run = ExamRun.get_currently_schedulable(course).first()
     if schedulable_exam_run is not None:
-        return schedulable_exam_run.date_last_schedulable.strftime("%B %-d")
+        return schedulable_exam_run.date_last_schedulable.strftime("%B %-d, %I:%M%p %Z")
```

Comment:
nit: can we add a space between the hour and the am/pm?

```suggestion
        return schedulable_exam_run.date_last_schedulable.strftime("%B %-d, %I:%M %p %Z")
```